src/System.Runtime.Extensions/src/System/IO/Path.Windows.cs

```diff
@@ -93,6 +84,15 @@ public static string GetFullPath(string path)
 
             return fullPath;
         }
+        
+        public static char[] GetInvalidFileNameChars() => new char[]
```

Comment:
Maybe this should be closer to the top of the file under the other constant/field definitions.